### ISSUE TITLE
steam game disconnects when workspace not visible

### ISSUE THREAD

**Issue by aeosynth** [conformal/spectrwm#141]
when i connect to a dota 2 game, if i switch workspaces while the game is loading, i end up disconnecting from the game. is spectrwm giving non-visible apps less resources somehow?

**Comment by LordReg** [comment]
Windows on hidden workspaces are put into an iconic state.  Maybe the game doesn't like being iconified while loading.  It may be hanging in a loop waiting for its window to be mapped only to have the connection time out.  I wouldn't be surprised if the Source 2 engine has some ICCCM and/or EWMH compliance issues given the issues I have discovered with Steam.  Are you running the game fullscreen or windowed?  Also, did you try reproducing the issue in any other window managers?

**Comment by aeosynth** [comment]
i'm running the game in 'desktop friendly fullscreen'. i tried bspwm and i3, they did not have this problem. is there a simple way to check if iconifying is the issue?

**Comment by LordReg** [comment]
I just tried dota2 and 'desktop friendly fullscreen' seems to work fine with 'Desktop-friendly fullscreen minimizes on focus loss' unchecked.  Also, I have 'Use my monitor's current resolution' selected.  I only tried watching a game and it connected fine, even when it was connecting while on a non-visible workspace or iconified using Mod-w.  Are you using different settings?  What version of spectrwm are you running?
